raft_test.go

```diff
@@ -204,14 +204,14 @@ func TestRaft_RecoverCluster(t *testing.T) {
 			c.fsms[i] = r2.fsm.(*MockFSM)
 		}
 		c.FullyConnect()
-		time.Sleep(c.propagateTimeout)
+		time.Sleep(c.propagateTimeout * 3)
 
 		// Let things settle and make sure we recovered.
 		c.EnsureLeader(t, c.Leader().localAddr)
 		c.EnsureSame(t)
 		c.EnsureSamePeers(t)
 	}
-	for applies := 0; applies < 20; applies++ {
+	for applies := 0; applies < 10; applies++ {
```

Comment:
We thought it would make sense to reduce this number because effectively this is just repeating the same test many times. We really only need to run this test twice, once with 0 applies , and once with n>1 applies, so 10 should be more than sufficient.